.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
```

Comment:
Probably. I just copied the default. Especially since the only test being run is Snyk which won't change between Node versions.